src/effect/transparent.rs

```diff
@@ -0,0 +1,35 @@
+use crate::{
+    audio_unit::{self, AudioUnit},
```

Comment:
Is `AudioUnit` unused? Is there linting warning for that?

Reply:
Yeah, unused imports will fail linting. It is used, the usage is just not obvious. `AudioUnit` is a trait, so it has to be imported to use any functions defined by that trait, so `self.unit.process(...)` requires it.

Reply:
Ah yes now that I'm catching up on your replies, you've explained this